share/lib/python/neuron/coreneuron.py

```diff
@@ -160,6 +172,8 @@ def nrncore_arg(self, tstop):
         # args derived from user properties
         if self._gpu:
             arg += " --gpu"
+            if self._num_gpus:
+                arg += " --num-gpus %d" % self._num_gpus
```

Comment:
We should rename this `if self._num_gpus` and always set the option, right?
Because `--num-gpus 0` means to use all GPUs